mobile/lib/ui/viewer/search_tab/people_section.dart

```diff
@@ -24,7 +26,7 @@ import "package:photos/utils/navigation_util.dart";
 
 class PeopleSection extends StatefulWidget {
   final SectionType sectionType = SectionType.face;
-  final List<SearchResult> examples;
+  final List<GenericSearchResult> examples;
```

Comment:
This is not required.
If you need the personID or clusterID, we should cast the search result as GenerateSearchResult and read the value from the params (not faceFilter)

Reply:
Using `List<GenericSearchResult>` is the best thing to do I think. All other section widgets do the same. 
I agree on the params suggestion.